### PR TITLE
Display users' avatar

### PR DESCRIPTION
**Pull request by Devessier**
Closes https://github.com/AdonisEnProvence/MusicRoom/issues/434

### CODE REVIEW

File: packages/client/components/kit/SvgImage.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { SvgImageProps } from './SvgImage.contract';
+
+const SvgImage: React.FC<SvgImageProps> = ({
+    uri,
+    accessibilityLabel,
+    style,
+}) => {
+    return (
+        <View style={style}>
+            <SvgUri uri={uri} accessibilityLabel={accessibilityLabel} />
```
Comment (Devessier) [inline packages/client/components/kit/SvgImage.tsx:13]: `SvgUri` component did not work from the first time on the Web, so I decided do use a usual `Image` component on the Web.

File: packages/client/components/User/UserListItem.tsx [hunk 2]
```diff
@@ -57,15 +57,14 @@ const UserListItem: React.FC<UserListItemProps> = (props) => {
                                     alignItems: 'center',
                                 }}
                             >
-                                <Image
-                                    source={{
-                                        uri: 'https://stately.ai/registry/machines/03107919-a451-4085-9b66-633cd8794164.png',
-                                    }}
-                                    sx={{
+                                <SvgImage
+                                    uri={`https://avatars.dicebear.com/api/big-smile/${props.user.userID}.svg`}
```
Comment (prastoin) [inline packages/client/components/User/UserListItem.tsx:61]: **👏 praise:** what a game changer ! great without any server implications

File: packages/client/components/kit/SvgImage.tsx (hunk 1 above)
Comment (prastoin) [reply packages/client/components/kit/SvgImage.tsx:13]: **👏 praise:** I think this a great idea

**Review by prastoin — APPROVED** [review]
Wow really cool !
**❔ question:** I was wondering if we would like to test the user avatar visibility/presence inside integ test nor e2e tests ?
Anyway from now I can already approve this PR

File: packages/client/e2e/profile/test-a.spec.ts [hunk 3]
```diff
@@ -274,34 +275,43 @@ test('Profile test-a', async ({ browser }) => {
...
 
-    //UserB looking on UserA profile page
+    //UserB looking at UserA profile page
     {
-        const allUserProfileInformationElements = userBPage.locator(
-            withinMyUserProfilePageContainer({
-                userID: userAUserID,
-                selector: 'css=[data-testid$="user-profile-information"]',
-            }),
+        const userAProfileScreen = withinUserProfileScreen({
```
Comment (prastoin) [inline packages/client/e2e/profile/test-a.spec.ts:280]: **praise:** syntax looks great

File: packages/client/e2e/_utils/mpe-e2e-utils.ts [hunk 4]
```diff
@@ -128,6 +128,16 @@ export function withinMyProfilePageContainer(selector: string): string {
     return `css=[data-testid="my-profile-settings-page-container"] >> ${selector}`;
 }
 
+export function withinUserProfileScreen({
```
Comment (prastoin) [inline packages/client/e2e/_utils/mpe-e2e-utils.ts:131]: **🎯 suggestion:** I understand that this method returns a locator. I think this great as it allow a better syntax further in the tests
But might the name change from previous usage ?
Just below `withinMyUserProfilePageContainer` returns the same selector as a string :thinking:

File: packages/client/screens/MyProfile/__tests__/MyProfile.test.tsx [hunk 5]
```diff
@@ -31,14 +31,21 @@ test('It should display my profile page with my profile information', async () =
 
     await waitFor(() => {
         expect(screen.getByTestId('my-profile-page-container')).toBeTruthy();
-        const devicesCounter = screen.getByText(/.*devices.*3/i);
-        const playlistsCounter = screen.getByText(/.*playlists.*4/i);
-        const followersCounter = screen.getByText(/.*followers.*5/i);
-        const followingCounter = screen.getByText(/.*following.*6/i);
+
```
Comment (prastoin) [inline packages/client/screens/MyProfile/__tests__/MyProfile.test.tsx:34]: **praise:** 👏

**Review by prastoin — APPROVED** [review]
Great work !
Leaved one suggestion above, can still already approve the PR
Ready to be merged

File: packages/client/e2e/_utils/mpe-e2e-utils.ts (hunk 4 above)
Comment (Devessier) [reply packages/client/e2e/_utils/mpe-e2e-utils.ts:131]: In fact I don't understand the name of `withinMyUserProfilePageContainer` function, as it does not return _my_ profile page, but the page of another user. I did not want to change `withinMyUserProfilePageContainer` behavior, so I duplicated it.

Comment (prastoin) [reply packages/client/e2e/_utils/mpe-e2e-utils.ts:131]: Ah yes :open_mouth: didn't read the name carefully
I guess it's a typo :thinking: 
Then no big deal we'll see later if we wanna change that

**Pull request merged**